install-nvm.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh
```

Comment:
if someone's put an install.sh file a PR will this overwrite it?  If not I guess it's not the end of the world cos we're verifying hash below

Reply:
Dunno. I just copy/pasted this from the workflow file.